interface/portraits/portraits_nightelf.gfx

```diff
@@ -11,36 +11,23 @@ spriteTypes = {
 		effectFile = "gfx/FX/portrait.lua"
```

Comment:
The beard goes behind the hood.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/82447812-d916d180-9ab9-11ea-8ee4-b7619dd43518.png)

</details>